_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/certificates.yml

```diff
@@ -48,6 +48,8 @@
         creates: cert-CNCIAgent-localhost.pem
       - role: controller
         creates: cert-Controller-localhost.pem
+      - role: agent,netagent
+        creates: cert-CNAgent-NetworkingAgent-localhost.pem
```

Comment:
I still prefer this to be conditional.  If there is no dual role machine, this needlessly creates an extra certificate which would provide elevated privileges to the holder.